atat/src/client.rs

```diff
@@ -77,8 +77,11 @@ where
             // `self.config.cmd_cooldown` ms have passed before sending a new
             // command
             block!(self.timer.wait()).ok();
-            for c in cmd.as_string().as_bytes() {
-                block!(self.tx.write(*c)).map_err(|_e| Error::Write)?;
+            let cmd_string = cmd.as_string();
+            #[cfg(feature = "logging")]
+            log::debug!("Sending command: {:?}", cmd_string.as_str());
```

Comment:
Could we make this a trace as well?

Reply:
The idea was to log incoming and outgoing (full) commands on debug level and low-level partial commands and byte parsing / state machine info on trace level. That way you can see what's happening on DEBUG and you can debug the parsing / states on TRACE.

Do you think that should all be on TRACE?